Firestore/Example/Tests/SpecTests/FSTSyncEngineTestDriver.h

```diff
@@ -246,10 +252,10 @@ typedef NSDictionary<FSTUser *, NSArray<FSTOutstandingWrite *> *> FSTOutstanding
  * sentWritesCount, but not necessarily, since the FSTRemoteStore limits the number of
  * outstanding writes to the backend at a given time.
  */
-@property(nonatomic, strong, readonly) FSTOutstandingWriteQueues *outstandingWrites;
+@property(nonatomic, assign, readonly) FSTOutstandingWriteQueues *outstandingWrites;
 
 /** The current user for the FSTSyncEngine; determines which mutation queue is active. */
-@property(nonatomic, strong, readonly) FSTUser *currentUser;
+@property(nonatomic, assign, readonly) firebase::firestore::auth::User *currentUser;
```

Comment:
Could this be a reference instead of a pointer? (Or does obj-c prefer pointers?)
Could this be const? (i.e. do callers have a need to mutate this?)

(also above)

Reply:
This can be a const reference because all callers are Objective-C++.